core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java

```diff
@@ -59,39 +66,51 @@ public void run() {
 
         @Override
         public void handleInput(int[] keys) {
-            if (disabled) {
-                for (int i : keys) {
-                    if (i == 'r') {
-                        promptHandler.setStatus(YELLOW + "Starting tests" + RESET);
-                        TestSupport.instance().get().start();
+            //common commands, work every time
+            for (int k : keys) {
+                if (k == 'h') {
+                    printUsage();
+                } else if (k == 'h') {
```

Comment:
Looks like the previous `if` already does that?

Reply:
oops